test/telemetry/src/main/java/org/opensearch/test/telemetry/tracing/StrictCheckSpanProcessor.java

```diff
@@ -60,7 +60,7 @@ private MockSpanData toMockSpanData(Span span) {
             System.nanoTime(),
             false,
             span.getSpanName(),
-            Thread.currentThread().getStackTrace(),
+            null,
```

Comment:
Please remove `stackTraceElement[] stackTrace` altogether, this was the only place it was used

Reply:
Putting it behind a flag so that it can be re-enabled if needed for debugging purposes.